includes/definitions/dell-powervault.yaml

```diff
@@ -0,0 +1,48 @@
+os: dell-powervault
+text: 'Dell PowerVault MD'
+type: storage
+icon: dell
+group: dell
+poller_modules:
```

Comment:
FYI, setting modules in an OS prevents users from setting them globally.
The poller_modules and discovery_modules is more for enabling required modules rather than disabling everything.

As I said before, feel free to set this in your config.php

Reply:
That's fine, it's only affects polling speed - I'll remove them.